packages/jellyfish-api-core/__tests__/category/loan/getInterest.test.ts

```diff
@@ -137,12 +137,12 @@ describe('Loan getInterest', () => {
     // calculate interest per block for TSLA
     const netInterest = (3 + 0) / 100 // (scheme.rate + loanToken.interest) / 100
     const blocksPerDay = (60 * 60 * 24) / (10 * 60) // 144 in regtest
-    const interestPerBlock = new BigNumber(netInterest).multipliedBy(1000).dividedBy(365 * blocksPerDay) //  netInterest * loan token amount(1000) / 365 * blocksPerDay
-    expect(interests[0].interestPerBlock.toFixed(8)).toStrictEqual(interestPerBlock.toFixed(8, 1))
+    const interestPerBlock = new BigNumber(netInterest * 1000 / (365.0 * blocksPerDay)).decimalPlaces(8, BigNumber.ROUND_CEIL) //  netInterest * loan token amount(1000) / 365 * blocksPerDay
+    expect(interests[0].interestPerBlock.toFixed(8)).toStrictEqual(interestPerBlock.toFixed(8)) // NOTE(sp): AIN use std::ceil(InterestPerBlockFloat()) after FCM hardfork, when storing the per block interest rate in DB
```

Comment:
wow thanks.. gonna update this cal to LW as well

Reply:
From `2.2.x` to `2.3.x` the calculation was drifted to this, not sure whether it's gonna drift again to something else for `2.4.x`... 🤔

Reply:
This change with some other changes was mainly added to address the original problem after FC.
Will check with Antoniy for any planned change of the calculation.

Reply:
https://github.com/DeFiCh/ain/issues/984